Testing/VelaptorTests/Reactables/Core/ReactorTests.cs

```diff
@@ -44,6 +44,22 @@ public void OnNext_WhenInvoked_ExecutesOnNext()
             Assert.True(onNextInvoked);
         }
 
+        [Fact]
+        public void OnNext_WhenCompleted_DoesNotExecutesOnNext()
```

Comment:
Refactor unit test method name to below:

```suggestion
        public void OnNext_WhenCompleted_DoesNotExecuteOnNext()
```